.pre-commit-config.yaml

```diff
@@ -0,0 +1,27 @@
+repos:
+
+- repo: https://github.com/pre-commit/pre-commit-hooks
+  rev: v4.1.0
+  hooks:
+  - id: check-added-large-files
+  - id: check-case-conflict
+  - id: check-yaml
+    args: ["--unsafe"]
+  - id: debug-statements
+  - id: end-of-file-fixer
+    exclude: ".*fits"
+  - id: trailing-whitespace
+    exclude: ".*fits"
```

Comment:
Do we need to exclude .asdf files here as well?

Reply:
No because those are counted as "binary" files somewhere.  Which means those actions do not get run on them.

Reply:
Ah!  Okay, great, thanks.